src/Panel/Panel/Panel.example.md

```diff
@@ -100,17 +100,21 @@ import Panel from '@terrestris/react-geo/Panel/Panel/Panel';
 </Panel>
 ```
 
+<!-- TODO: for some reason, x and y needs to be defined when width and height are given 
+  or else it will a traansform class with a traanslaate() method will be passed to the style (e.g. transform: translate(523px, 584px))
+  With the latest update of react-rnd 10.2.1 to 10.3.4 this issue persists which probaably means the errors is coming from the Panel Component. Adding the x,y is a workaround for now, but this issue should be addressed in another PR 
+-->
```

Comment:
```suggestion
<!-- TODO: for some reason, x and y needs to be defined when width and height are given 
  or else it will a transform class with a translate() method will be passed to the style (e.g. transform: translate(523px, 584px))
  With the latest update of react-rnd 10.2.1 to 10.3.4 this issue persists which probably means the error is coming from the Panel Component. Adding the x,y is a workaround for now, but this issue should be addressed in another PR.
-->
```